_docs-sources/foundations/iac-foundations/labels-tags.md

```diff
@@ -0,0 +1,264 @@
+
+## Tags Overview
+
+Tags are key value pairs that act as metadata for organizing your AWS resources. Tags are often used by operators and developers to identify workload owners (e.g., a specific team), the application that a resource belongs to, the environment in which an application is running, and more. Finance teams often make use of teams to map cost and usage to specific Cost Centers or Business Units. For more information on tags, see the [official documentation](https://docs.aws.amazon.com/tag-editor/latest/userguide/tagging.html).
+
+### Tag key recommendations
+
+Gruntwork recommends prefixing all custom tag keys with the name of your organization. For example - the company acme co would start all of their tag keys with `acmeco`. This will allow you to distinguish tags specific to your organization from tags that may be automatically applied to your by AWS.
+
+### Recommended tags
+
+The table below contains recommend tag keys and the description for the value you should give to the key. These keys are beneficial for both locating and filtering for resources as an operator or developer as well as being used as [cost allocation tags](https://docs.aws.amazon.com/awsaccountbilling/latest/aboutv2/cost-alloc-tags.html) to assist finance teams with associating costs to specific business units or applications.
+
+
+| Key Name   | Example Key | Value Description |
+| ---------- | ----------- | ----------- |
+| Department | `acmeco:Department` | The organizational department to which the owning team belongs. Useful for rolling up costs per department in the organization |
+| Team | `acmeco:Team` | The team that owns the resource. Useful for tracking resources and cost per team. Useful when combined with department tags to see how much each team in a department is spending |
+| Environment | `acmeco:Environment` | The environment in which the resource is running. Useful for tracking spend per environment at the team, business unit, and organization levels |
+| Application | `acmeco:Application` | Track spend per application. Can be used in conjunction with environment tags to see cost of running an application per environment |
+| CostCenter | `acmeco:CostCenter` | Monitor costs by cost center |
+| BusinessUnit | `acmeco:BusinessUnit` | Monitor costs by business unit |
+| Workload Type | `acmeco:WorkloadType` | Course grained type of service, e.g., compute, data storage, databases, etc |
+
+### Cost Allocation Tags
+
+If you are using custom tags for your resources for the purposes of tracking cost and usage, you will need to activate the tags in the Billing and Cost Management console in AWS. For more information see [User-defined cost allocation tags](https://docs.aws.amazon.com/awsaccountbilling/latest/aboutv2/custom-tags.html) and [Activating user-defined cost allocation tags](https://docs.aws.amazon.com/awsaccountbilling/latest/aboutv2/activating-tags.html).
```

Comment:
Link to the "Billing and Cost Management" console?

Reply:
Those links are a little unintuitive because if you aren't already logged in to the console then it will just go to either the log in page, then when you log in you don't get forward to your intended page iirc. Happy to be overruled here if you think it would be helpful.